fabriceumuhire/MY-BRAND
# Commit of Blog feature code

### What the PR does
- CRUD articles
### Tasks description
- Create article using POST method and store the image to cloudinary 
- Retrieve articles or one article by id from the DB
- Update article and store in the DB
- Delete article from DB and image from Cloudinary
### How to test
- Use POSTMAN or MongoDB Compass to view all queries
- Check URL from the article if they display images.
